### ISSUE TITLE
ecoinvent downloader does not list databases anymore

### ISSUE THREAD

**Issue by bsteubing** [LCA-ActivityBrowser/activity-browser#649]
![image](https://user-images.githubusercontent.com/33026150/134649891-d4f07d0c-78f8-422a-861f-b75c480e8453.png)

**Comment by haasad** [comment]
Hi @bsteubing and @StpdFox 
I added a fix to eidl, if you either `conda update activity-browser-dev` or `conda update eidl` it should work again.
I also got rid of some of the invalid options (reports etc.) and the `cut-off`/`cutoff` options that were available before.
Please let me know if anything doesn't work.

Cheers,
Adrian

**Comment by StpdFox** [comment]
Thanks for the quick response! I'm going to test your changed right away.

**Comment by StpdFox** [comment]
I'm able to see the list of available ecoinvent databases again, but i'm met with another error, which is new for me. 

```
ERROR: No more files
/Users/vosm4/Library/Application



System ERROR:
Unknown error: -2147024872
```

![image](https://user-images.githubusercontent.com/8734631/134876741-8f3f4c6d-1397-46fb-bf5c-401adf3c597c.png)


 Not sure what is causing this.

**Comment by haasad** [comment]
Could you maybe just delete the offending file? Eidl stores previously downloaded ecoinvent files for offline use and to speed up the process, so you don't always have to re-download it. Maybe it's really just an older corrupted file that you have laying around.

**Comment by StpdFox** [comment]
Oh i forgot to add, i have removed it, but it's actually calling the file it just downloaded corrupt it seems? 
It could be a check on the AB side, which i'm trying to find now.

**Comment by haasad** [comment]
Works for me both in AB and with eidl only. It could potentially be a Mac issue.

This error could indicate that something trips on the space in the path, ie. `Application Support`:
```
ERROR: No more files
/Users/vosm4/Library/Application
```

**Comment by haasad** [comment]
But I don't see how this could be related to any of my changes if it worked previously

**Comment by StpdFox** [comment]
True, I'll try to identify the issue on my machine, and hope it only applies to mine. Thanks for the quick fix anyways :)

**Comment by bsteubing** [comment]
Hi @haasad, sorry to bother you again, but @StpdFox reported yesterday that there is still an issue with the SSL certificate for downloading ecoinvent databases with the eidl. "There's another issue with the downloader, where we would need to update the SSL certificate to be able to download with the ecoinvent downloader. Also, on my windows machine i'm unable to open a local downloaded ecoinvent database. I'm get the same 'archive is corrupted' error as we had before... ". I am unfortunately not really familiar with this part of the code, do you know what needs to be done here?

**Comment by haasad** [comment]
Hey @bsteubing, I'll have a look into the archive corrupted issue. I couldn't reproduce it on osx or linux. I'll check on windows when I find the time.

Regarding the SSL certs, this is neither an activity browser nor an eidl issue:
```
curl https://ecoquery.ecoinvent.org 
curl: (60) SSL certificate problem: unable to get local issuer certificate
```
This is related to the expiration of a root certificate used by let's encrypt, which expired yesterday:
https://letsencrypt.org/docs/dst-root-ca-x3-expiration-september-2021/

```
openssl s_client -connect ecoquery.ecoinvent.org:443 -servername ecoquery.ecoinvent.org
CONNECTED(00000003)
depth=2 O = Digital Signature Trust Co., CN = DST Root CA X3
verify error:num=10:certificate has expired
notAfter=Sep 30 14:01:15 2021 GMT
verify return:1
depth=2 O = Digital Signature Trust Co., CN = DST Root CA X3
notAfter=Sep 30 14:01:15 2021 GMT
verify return:1
depth=1 C = US, O = Let's Encrypt, CN = R3
verify error:num=10:certificate has expired
notAfter=Sep 29 19:21:40 2021 GMT
verify return:1
depth=1 C = US, O = Let's Encrypt, CN = R3
notAfter=Sep 29 19:21:40 2021 GMT
verify return:1
depth=0 CN = a.eiservers.ifu.com
notAfter=Dec  7 11:50:49 2021 GMT
verify return:1
---
Certificate chain
 0 s:CN = a.eiservers.ifu.com
   i:C = US, O = Let's Encrypt, CN = R3
 1 s:C = US, O = Let's Encrypt, CN = R3
   i:O = Digital Signature Trust Co., CN = DST Root CA X3
---
```

**Comment by StpdFox** [comment]
It does work when you manually extract the database and import from directory. So there is a workaround

**Comment by bsteubing** [comment]
@haasad question: can we also download the new cut-off EN 15804 database version from ecoinvent 3.8 using the EIDL?
I copy @cmutel as we were just discussion this...

**Comment by haasad** [comment]
@bsteubing I can take a look. Would you mind opening an issue in https://github.com/haasad/EcoInventDownLoader so this doesn't get lost?
